backend/compact-connect/lambdas/purchases/tests/function/test_handlers/test_privilege_options.py

```diff
@@ -0,0 +1,96 @@
+import json
+
+from moto import mock_aws
+from tests.function import TstFunction
+
+TEST_COMPACT = "aslp"
+TEST_PROVIDER_ID = "some-provider-id"
+@mock_aws
+class TestGetPurchasePrivilegeOptions(TstFunction):
+
+    def _when_testing_provider_user_event_with_custom_claims(self, test_compact=TEST_COMPACT):
+        self._load_compact_configuration_data()
+        with open('tests/resources/api-event.json', 'r') as f:
+            event = json.load(f)
+            event['requestContext']['authorizer']['claims']['custom:providerId'] = TEST_PROVIDER_ID
+            event['requestContext']['authorizer']['claims']['custom:compact'] = test_compact
+
+        return event
+
+    def test_get_purchase_privilege_options_returns_expected_jurisdiction_option(self):
+        from handlers.privileges import get_purchase_privilege_options
+        event = self._when_testing_provider_user_event_with_custom_claims()
+
+        resp = get_purchase_privilege_options(event, self.mock_context)
+
+        self.assertEqual(200, resp['statusCode'])
+        privilege_options = json.loads(resp['body'])
+
+        with open('tests/resources/dynamo/jurisdiction.json', 'r') as f:
+            expected_jurisdiction_option = json.load(f)
+            expected_jurisdiction_option.pop('pk')
+            expected_jurisdiction_option.pop('sk')
+            # we should not be returning email information in the response
+            expected_jurisdiction_option.pop('jurisdictionOperationsTeamEmails')
+            expected_jurisdiction_option.pop('jurisdictionAdverseActionsNotificationEmails')
+            expected_jurisdiction_option.pop('jurisdictionSummaryReportNotificationEmails')
+            # remove date fields as they are not needed in the response
+            expected_jurisdiction_option.pop('dateOfUpdate')
+
+
+        # the jurisdiction configuration is stored in the dynamo db as part of the
+        # parent TstFunction setup, so we can compare the response directly
+        jurisdiction_option = [option for option in privilege_options['items'] if option['type'] == 'jurisdiction'][0]
```

Comment:
should we test that this list is only of length 1 before pulling out the first element?

Reply:
added that check